doc-content/enterprise-only/installation/clustering-kie-server-failover-proc.adoc

```diff
@@ -0,0 +1,106 @@
+[id='clustering-kie-server-failover-proc_{context}']
+= Failover for jobs in the RUNNING state on failed {KIE_SERVER} nodes
```

Comment:
Needs to be changed to a procedure description. How about:
Configuring failover for jobs in the RUNNING state on failed {KIE_SERVER} nodes

Reply:
Wow, great catch!